modules/n1ql/pages/n1ql-language-reference/numericfun.adoc

```diff
@@ -4,102 +4,127 @@
 
 {description}
 
-ABS(expression)
+== ABS(expression)
 
+=== Description
 Returns absolute value of the number.
 
-ACOS(expression)
+== ACOS(expression)
 
-Returns arccosine in radians.
+=== Description
+Returns arc cosine in radians.
 
-ASIN(expression)
+== ASIN(expression)
 
-Returns arcsine in radians.
+=== Description
+Returns arc sine in radians.
 
-ATAN(expression)
+== ATAN(expression)
 
-Returns arctangent in radians.
+=== Description
```

Comment:
Even though it is a well-known function, it would be great if we could add usage example per function